services/can_logger/can_logger.py

```diff
@@ -8,51 +7,124 @@
 import sys
 import collections
 import multiprocessing as mp
+import logging
 from datetime import datetime
 from psycopg2 import OperationalError
 from psycopg2.extras import execute_values
 
-
 def write_to_csv(wr_buff, can_bus):
```

Comment:
We open the log file for every message written?

Reply:
No. We open every time the write_buffer is full (which right now stores up to 1000 messages). So depending on the message rate, the file will be more of less frequently opened.